readme.md

```diff
@@ -4,15 +4,48 @@
 
 ## Install (coming soon)
 
-[link-chrome]: https://chrome.google.com/webstore/detail/ 'Version published on Chrome Web Store'
+## Chrome
+
+1. [Download Chrome Extension Zip](https://on-deck-community.enterprise.slack.com/files/U019MVC9TEW/F020MH820Q3/distribution.zip)
```

Comment:
Update to "https://github.com/zlwaterfield/refined-zapier/releases/download/0.0.1/refined-zapier-0.0.1.zip"